examples/Polls API.md

```diff
@@ -80,12 +80,12 @@ This action allows you to vote on a question's choice.
 
             Location: /questions/1
 
-## Questions collection [/questions{?page}]
+## Questions Collection [/questions{?page}]
 
 + Parameters
     + page (optional, number, `1`) ... The page of questions to return
 
-### List all questions [GET]
+### List All Questions [GET]
```

Comment:
Are we sure about this? In apiary docs, this is an action name and this would still be title case.